pkg/controller/federatedcluster/clusterclient.go

```diff
@@ -147,8 +146,7 @@ func getZoneNameForNode(node corev1.Node) (string, error) {
 			return value, nil
 		}
 	}
-	return "", errors.Errorf("Zone name for node %s not found. No label with key %s",
-		node.Name, LabelZoneFailureDomain)
+	return "", nil
```

Comment:
Why is it desirable to remove detail from these messages?

cc: @shashidharatd

Reply:
because this message is shown on every iterations and it's too verbose IMO.

Reply:
I think we need to find a way to reduce the incidence of logging, then, rather than removing it entirely.  My understanding is that this will be useful information for anyone trying to troubleshoot federated dns or ingress.

Reply:
ok. how about having a "already-warned" flag in ClusterClient and make it warn only the first time?

Reply:
Do agree with @marun, we need to reduce the incidence of logging. But at the same time, when one is not using the federated dns feature, this error should not be overwhelming the logs. I don't have an elegant solution for this but maybe we could keep a variable in controller per cluster to say, error is already reported.

Reply:
i've pushed new patches to do that.